City-of-Helsinki/berth-reservations-admin
# VEN-501 | Fetch customer boats

## Description :sparkles:
- Fetch the customer's boats from the backend and replace the mock data on the individual customer page

## Issues :bug:

### Closes :no_good_woman:

### Related :handshake:

## Testing :alembic:

### Automated tests :gear:️
- https://github.com/City-of-Helsinki/berth-reservations-admin/blob/6d5a8ecf337e3babac714826219b622867749ccc/src/domain/individualCustomer/utils.test.js#L36

### Manual testing :construction_worker_man:
- From the sidebar click on `Asiakkaat`
- Click on one of the customers in the table
- Scroll down on the single customer page to see the customer's boats under `ASIAKKAAN VENEET` card

Note: This [customer](https://venepaikka-admin.test.kuva.hel.ninja/customers/UHJvZmlsZU5vZGU6N2QyMGZlNmMtYWMzZC00ZTY0LWJkYzAtYTJiZjg5ODU5ZGU3) has a large boat.

## Screenshots :camera_flash:
- No boats:
![image](https://user-images.githubusercontent.com/23040926/78016089-02856d80-7353-11ea-84bc-ab094bf2b614.png)

- One boat:
![image](https://user-images.githubusercontent.com/23040926/78015841-a3bff400-7352-11ea-920f-704ea21f1759.png)

- One large boat:
![image](https://user-images.githubusercontent.com/23040926/78016041-f00b3400-7352-11ea-8a5c-12caaa9276eb.png)

## Additional notes :spiral_notepad:
If the customer has more than one boat they will be listed below each other
